Ds3/Helpers/Strategies/ChunkStrategies/ReadRandomAccessChunkStrategy.cs

```diff
@@ -103,6 +106,20 @@ private TransferItem[] GetNextTransfers()
             .GetJobChunksReadyForClientProcessingSpectraS3(new GetJobChunksReadyForClientProcessingSpectraS3Request(this._jobId))
             .Match((ts, jobResponse) =>
             {
+                if (_lastAvailableChunks == null)
```

Comment:
Rather than having to blocks in this if chain both having the same code, add a null check to the `else if` statement at the beginning, remove this first if block, and leave the last block.  So:

```csharp
if (_lastAvailableChunks != null && GotTheSameChunks(_lastAvailableChunks, GetChunksNumbers(jobResponse)))
{
    this.SameChunksRetryAfter.RetryAfterFunc(ts);
    return new TransferItem[0];
}
else
{
    _lastAvailableChunks = GetChunksNumbers(jobResponse);
}
```